standard-hackathon-rules.md

```diff
@@ -10,14 +10,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 ## The rules of the competition
 
-1. There is no maximum or minimum team size. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
+1. There is no maximum or minimum team size, however we do recommending having up to 4 team members to maximize the hackathon experience. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
 2. Teams should be made up exclusively of students (or recent graduates within one year of having graduated) who are not organizers, volunteers, judges, sponsors, or in any other privileged position at the event. 
 3. All team members should be present at the event. Leaving the venue for some time to hack elsewhere is fine.
 4. Teams can of course gain advice and support from organizers, volunteers, sponsors, and others.
-5. All work on a project should be done at the hackathon.
-6. Teams can use an idea they had before the event.
+5. All work on a project that will be demo'd should only have been created at the current hackathon.
+6. Teams can use an idea they had before the event.  Code built prior to the event can be used but must be properly documented.  Only new code will be considered eligible at the current hackathon.  
```

Comment:
"Code wrote" instead of built

Reply:
"Code written prior to the event must be open source" -- look at how other hackathons have written their rules. If everything is open source + available to every other attendee is fine, but if it's closed code, then that's not OK.